de.dlr.sc.virsat.graphiti.test/src/de/dlr/sc/virsat/graphiti/test/AllTests.java

```diff
@@ -19,20 +19,26 @@
 import junit.framework.JUnit4TestAdapter;
 
 
+
+
+/**
+ *
+ */
 @RunWith(Suite.class)
 
-@SuiteClasses({
-	BeanIndependenceSolverTest.class,
-	DiagramHelperTest.class,

```

Comment:
Since we arent registering any test cases here, I dont think this file should have changes.